addon/templates/components/-ember-popper-legacy.hbs

```diff
@@ -0,0 +1,10 @@
+{{yield (hash
+  popper=(hash
```

Comment:
@pzuraq you open to dropping the nested popper hash once able? I'm down to keep it in for now, but would love to see the yielded hash simplified in the future.